src/LuaPrinter.ts

```diff
@@ -742,11 +742,15 @@ export class LuaPrinter {
     public printMethodCallExpression(expression: lua.MethodCallExpression): SourceNode {
         const chunks = [];
 
-        const prefix = this.printExpression(expression.prefixExpression);
-
         const name = this.printIdentifier(expression.name);
 
-        chunks.push(prefix, ":", name, "(");
+        const prefix = this.printExpression(expression.prefixExpression);
```

Comment:
How about something like that to remove parenthesizing duplication? 
```suggestion
        const prefix = this.printExpression(lua.isStringLiteral(expression.prefixExpression) ? expression.prefixExpression : lua.createParenthesizedExpression(expression.prefixExpression));;
```

Also, what about `"foo".bar`?

Reply:
Should not be affected by this PR, I'd prefer to keep that for the full parenthesis to printer change.